### ISSUE TITLE
Thoughts on changing the diffusion sampler ?

### ISSUE THREAD

**Issue by Ryu1845** [neonbjb/tortoise-tts#262]
Reading the paper draft, I noticed you were using DDIM with a linear schedule. Do you think changing the sampler to dpm-solver++ would improve quality and lower the number of inference steps, like it did for Stable Diffusion? Maybe the schedule could be swapped for Karras' but I think that incurs a slower inference time so that might not be for the best.

**Comment by neonbjb** [comment]
Yes, I bet that would help! Would be happy to review PRs that add this!
